Dockerfile

```diff
@@ -42,5 +42,8 @@ RUN chown -R siggame:siggame $HOME/ophelia/*
 # Change back to the siggame user
 USER siggame
 
+# Run setup scripts
+CMD [ "npm", "run", "setup"]
```

Comment:
Typically there should be only one `CMD`, you should RUN npm run setup and leave the `CMD` below as is